### DISCUSSION TITLE
Write a blog about Dapr.io

### DISCUSSION THREAD

**Discussion by ghost** [dragonflydb/dragonfly#2063] [Ideas]
https://dapr.io - is a very easy way to build microservices, pub-sub, store, actors, etc. all-in-one solution, vendor-agnostic.

They support Redis for pub-sub and store, I do use them with DragonFly now and works flawlessly for weeks now.

Maybe it's worth to write a blog about DragonFly using with Dapr instead of Redis?

![image](https://github.com/dragonflydb/dragonfly/assets/74614193/0a5e903a-d2a7-40f1-b32a-80f1b75e5a1c)

**Comment by romange** [comment]
Yes, that would be very cool!  @Niennienzz fyi.

@grubeli can you tell us what are you building with dapr?

**Reply by ghost** [reply]
In my backend there are several workers processing requests (image optimization, doesn't matter), when an user requests processing the gateway puts the requests to queue via Dapr API, and the subscribers (workers) getting them via Dapr API as well, that's Dapr's pubsub component, for the backend using Redis (now DragonFly) there are many backend options, rabbitMQ, Kafka, etc. DragonFly is good for me, simple and works.
Another component I'm using is store, it's kinda key-value solution abstracted by Dapr, also backed by DragonFly/Redis

There is also `Distributed Lock` also backed by Redis/DragonFly which can be used to access shared resources (didn't try yet).

There many components  and backends: https://docs.dapr.io/concepts/components-concept/

![image](https://github.com/dragonflydb/dragonfly/assets/74614193/f8a4feb2-8677-40ee-b2fc-0fc54dbf8c82)

**Comment by Niennienzz** [comment]
This are very interesting use cases. 😲  

We have a few lined up blog posts, we should try to plan/include a Dapr post.

**Comment by romange** [comment]
This guy has deleted his github account 🤷🏼
